src/pipx/constants.py

```diff
@@ -49,13 +53,22 @@ def is_mingw() -> bool:
     eval "$(register-python-argcomplete pipx)"
 
 zsh:
-    To activate completions for zsh you need to have
-    bashcompinit enabled in zsh:
+    To activate completions in zsh, first make sure compinit is marked for
+    autoload and run autoload:
+
+    autoload -U compinit && compinit
+
+    Afterwards you can enable completion for pipx:
+
+    eval "$(register-python-argcomplete pipx)"
+
+    NOTE: If your version of argcomplete is earlier than v3, you may need to
+    have bashcompinit enabled in zsh by running:
 
     autoload -U bashcompinit
     bashcompinit
```

Comment:
This should look a bit better.

```suggestion
    > [!NOTE]
    >
    > If your version of argcomplete is earlier than v3, you may need to
    have bashcompinit enabled in zsh by running:
    >
    > autoload -U bashcompinit
    > bashcompinit
```

Reply:
Is that a special syntax or stylistic convention? I don't believe I'm familiar with it.